layouts/_default/_markup/render-passthrough.html

```diff
@@ -0,0 +1,9 @@
+{{- $opts := dict "output" "htmlAndMathml" "displayMode" (eq .Type "block") }}
+{{- with try (transform.ToMath .Inner $opts) }}
```

Comment:
@catap 
Which version of Hugo are you using?
I suspect the Hugo version I'm running on Vercel is too old to recognize this `try` function.

Reply:
I think it needs at least [v0.132.0](https://github.com/gohugoio/hugo/releases/tag/v0.132.0).

But it was released 1,5 year ago...

Reply:
Ok, `try` needs [v0.141.0](https://github.com/gohugoio/hugo/releases/tag/v0.141.0) and base on [Repology](https://repology.org/project/hugo-sitegen/versions) I think it safe to use it:

![Packaging status](https://repology.org/badge/vertical-allrepos/hugo-sitegen.svg)](https://repology.org/project/hugo-sitegen/versions)